src/Microsoft.Health.Fhir.Core/Features/Conformance/IProvideCapability.cs

```diff
@@ -14,6 +16,6 @@ public interface IProvideCapability
         /// Allows this component to configure the server's CapabilityStatement.
         /// </summary>
         /// <param name="builder">Instance of the CapabilityStatementBuilder.</param>
-        void Build(ICapabilityStatementBuilder builder);
+        Task Build(ICapabilityStatementBuilder builder);
```

Comment:
Is it possible to rename this method to BuildAsync and expect a cancellation token?